src/character.h

```diff
@@ -1283,9 +1283,10 @@ class Character : public Creature, public visitable
         // Return all the limb special attacks the character has, if the parent limb isn't too encumbered
         std::set<matec_id> get_limb_techs() const;
         int get_working_arm_count() const;
+        /** Returns true if at least half your legs are working */
```

Comment:
However, if I understood correctly, `return working_limb_count == limb_count;` in `enough_working_legs()` means all legs are working.

Reply:
Thanks!

Reply:
Thanks for the review 😄